qiskit_ibm_runtime/qiskit_runtime_service.py

```diff
@@ -1183,7 +1330,7 @@ def instances(self) -> List[str]:
         """
         if self._channel == "ibm_quantum":
             return list(self._hgps.keys())
-        return []
+        return [t["crn"] for t in self._all_instances]
```

Comment:
This line returns an empty list for the case where no instance is given, but I would expect it to list all available instances, as when calling `list_instances()`